File: app/src/main/java/no/nav/tilbakemeldingsmottak/consumer/saf/journalpost/SafJsonJournalpost.java
package no.nav.tilbakemeldingsmottak.consumer.saf.journalpost;

import java.io.Serializable;

public class SafJsonJournalpost implements Serializable {

    private DataJournalpost data;

    public DataJournalpost getData() {
        return data;
    }

    public void setData(DataJournalpost data) {
        this.data = data;
    }

    public SafJournalpostTo getJournalpost() {
        return data.getJournalpost();
    }
}

File: app/src/main/kotlin/no/nav/tilbakemeldingsmottak/consumer/saf/journalpost/SafJsonJournalpost.kt
package no.nav.tilbakemeldingsmottak.consumer.saf.journalpost

import java.io.Serializable
import no.nav.tilbakemeldingsmottak.saf.generated.hentjournalpost.Journalpost

class SafJsonJournalpost : Serializable {
    var data: DataJournalpost? = null
    val journalpost: Journalpost?
        get() = data?.journalpost
}
